### PR TITLE
Adds cursor pointer for a tag

### PR DESCRIPTION
**Pull request by realsuiciniv**
This PR adds css `cursor:pointer` for all  `a` elements.
This will fix some usability problems.

Adds `.idea/` folder to `.gitignore` also.

### CODE REVIEW

**Comment by rogeriochaves** [comment]
Thank you for the PR!

But actually, I believe this is the wrong way to fix this, it is the default behaviour for the browser to add the pointer cursor for every link with a `<a href="something" />`, if some link is not showing the pointer then we have to fix the `a` element and add the `href` to it (which is the case for the edit profile menu link).

If we cannot add a `href`, then it shouldn't be a link in the first place, instead, it should actually be a button (like the logout menu button), which will also have the pointer.

To summarize, I believe this should not be fixed with CSS, but with correct semantics, which will be better for accessibility and anything else that depends on browser's default behaviour.

I'd say to leave without this CSS rule to leave us in an uncomfortable place and force us to fix it.

**Comment by realsuiciniv** [comment]
Makes sense.
So I will close this PR and open an Issue just to track this  problem 👍
